### DISCUSSION TITLE
Nova Buttons

### DISCUSSION THREAD

**Discussion by illiachernieha** [laravel/nova-issues#5248] [Q&A]
Hi all.
Is there a possibility to create a new button on the tool bar (see screenshot) with custom name, functionallity, modal pop up etc.
 
![image](https://user-images.githubusercontent.com/95935416/213139325-3cafb255-e538-43fc-bb07-a1a2de609758.png)

**Comment by davidhemphill** [comment]
What kind of API would you expect?

**Reply by illiachernieha** [reply]
Hello and thank you for your reply 
I'm wishing that I could have a custom button created on a specific resource that will allow me to execute custom code and user intefrace.
For example we want to create multiple contact ceration in one go, this requires custom code and we need a button in Nova user interface to fire off the workflow.

This is an example of a custom page that we want to create when we're firing off the button
https://5er11p.axshare.com/#g=1&p=create_multiple_contacts

P.S. Nova has been updated to version 4

**Reply by illiachernieha** [reply]
Also, I can see that other users are asking the same feature 
https://stackoverflow.com/questions/75154501/how-to-create-a-custom-button-in-laravel-nova-that-does-custom-function
